test/test_KernelTensors_correctness.jl

```diff
@@ -89,6 +89,6 @@ c = x -> 0.1 - 0.23*x[1] - 0.23*x[2] +  0.2*x[1]*x[2]
 c = x -> 0.1 - 0.23*x[1] - 0.23*x[2] +  0.2*x[1]*x[2]
 @test polyfit(c, 1, 10000.0) ≈ [0.1 -0.23; -0.23 0.2] rtol=rtol
```

Comment:
Also need to add unit tests in the Sources_correctness.jl for the new source term